bin/chef-zero

```diff
@@ -93,7 +93,11 @@ if options[:daemon]
     Process.daemon(true)
     server.start(true)
   else
-    abort 'Process.daemon requires Ruby >= 1.9'
```

Comment:
Agreed. This results in a confusing error when run on Windows. The daemon method is not supported by the Windows build of Ruby and will fail this check. A more descriptive error message is nice.